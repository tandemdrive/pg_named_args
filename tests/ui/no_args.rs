use pg_named_args::query_args;

fn main() {
    query_args!("$a, $b");
    query_args!("${a}");
}
