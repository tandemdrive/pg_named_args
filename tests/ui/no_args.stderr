error: expected `Args` struct
 --> tests/ui/no_args.rs:4:5
  |
4 |     query_args!("$a, $b");
  |     ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `query_args` (in Nightly builds, run with -Z macro-backtrace for more info)
